python/hsfs/core/feature_group_base_engine.py

```diff
@@ -52,3 +52,14 @@ def update_statistics_config(self, feature_group):
         self._feature_group_api.update_metadata(
             feature_group, feature_group, "updateStatsConfig"
         )
+
+    def update_config(self, feature_group, metadata):
+        """Update the metadata attribute specified of the feature group ."""
+        metadata_values = {
+            "updateStatsSettings": True,
+            "validationType": feature_group.validation_type,
+        }
```

Comment:
this looks like old code to me, I actually think we forgot to remove the function previously, cause look at line 50 above.

Reply:
It's still used here https://github.com/logicalclocks/feature-store-api/blob/ee70fecba586727d5ae9e5cc1d2e10463fad9819/python/hsfs/feature_group.py#L1224 

do you mean we should have an update_validation function?

Reply:
I will try to use the update_metadata function

Reply:
fixed